serverless/develop/serverless-kafka-developer.adoc

```diff
@@ -0,0 +1,41 @@
+[id="serverless-kafka-developer"]
+= Knative Kafka
+include::modules/serverless-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: serverless-kafka-developer
+
+toc::[]
+
+Knative Kafka functionality is available in an {ServerlessProductName} installation xref:../../serverless/admin_guide/serverless-kafka-admin.adoc#serverless-kafka-admin[if a cluster administrator has installed the `KnativeKafka` custom resource].
```

Comment:
The [ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#f) has guidelines for the use of _functionality_. Since I am not very familiar with Knative Kafka, I don't feel qualified to assess if you could substitue _functions_ here and keep the intended meaning.

Reply:
I don't think so, plus the fact that we have something called _Functions_ as a component of Serverless, so I think this would make it too confusing 😄 I think the ISG description actually fits here though; Knative Kafka refers to a handful of different "functions" and the things that can be configured etc for them (i.e. Kafka channels, brokers and event sources, and eventually sinks).